editor/modes/visual-editor/block-list.js

```diff
@@ -218,7 +218,7 @@ class VisualEditorBlockList extends Component {
 						onSelectionStart={ this.onSelectionStart }
 						onShiftSelection={ this.onShiftSelection }
 					/>,
-					<VisualEditorSiblingInserter
+					index < blocks.length - 1 && <VisualEditorSiblingInserter
```

Comment:
Lack of parentheses make this a bit ambiguous (similar to [`no-mixed-operators`](https://eslint.org/docs/rules/no-mixed-operators)). Is `<` considered before or after the subtraction (`index < blocks.length`)? Is subtraction including the element (`1 && <VisualEditorSiblingInserter`)?

Maybe:

```jsx
index < ( blocks-length - 1 ) && (
	<VisualEditorSiblingInserter
```

I have some similar thoughts here as in https://github.com/WordPress/gutenberg/pull/3246#pullrequestreview-72902056 between this approach and allowing the inserter component itself to consider whether it should render itself. Any thoughts? I don't feel strongly either way.

Reply:
The other approach seems fine to me.